src/ui/components/filters/geolocationcomponent.js

```diff
@@ -217,11 +238,20 @@ export default class GeoLocationComponent extends Component {
           this.core.persistentStorage.delete(`${StorageKeys.QUERY}.${this.name}`);
           this.core.persistentStorage.delete(`${StorageKeys.FILTER}.${this.name}`);
         },
-        () => this.setState({ geoError: true })
+        () => this._handleGeolocationError(),
+        this._geolocationOptions
       );
     }
   }
 
+  _handleGeolocationError () {
+    this.setState({ geoError: true });
```

Comment:
What behavior is triggered by geoError true? There's not already some kind of UI notification is there?

Reply:
yup there is the GeolocationFilter will display whatever it's error text config is set to, defaulting to 'Unable To Use Location'